### DISCUSSION TITLE
Integrate Custom Editors or Webview with LSP (Language Server Protocol)

### DISCUSSION THREAD

**Discussion by leowonglaw** [microsoft/vscode-discussions#754] [Extension Development QnA]
In case of having a Monaco Editor on a Webview itself or on a Custom Editor, how could it use LSP so it has intelliSense capabilities?

On this discussion https://github.com/microsoft/vscode-discussions/discussions/74#discussioncomment-3309227 it suggest to proxy all LSP methods, is there an easy way to directly integrate to the server?

**Comment by isidorn** [comment]
I will go ahead and close this discussion as a duplicate of the one you linked.
I think that is still the recommendation, and I am not sure if there is a better way to achieve this.
